constructor/osx/post_extract.sh

```diff
@@ -81,7 +81,7 @@ done
 
 # Cleanup!
 rm -f "$CONDA_EXEC"
-find "$PREFIX/pkgs" -type d -empty -exec rmdir {} \; 2>/dev/null || :
+find "$PREFIX/pkgs" -type d -empty -exec rmdir "{}" \; 2>/dev/null || :
```

Comment:
```suggestion
find "$PREFIX/pkgs" -type d -empty -exec rmdir {} \; 2>/dev/null || :
```

Oops, oversight on my end.